packages/contracts-bedrock/contracts/test/CrossDomainMessenger.t.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Messenger_Initializer } from "./CommonTest.t.sol";
+
+// CrossDomainMessenger_Test is for testing functionality which is common to both the L1 and L2
+// CrossDomainMessenger contracts. For simplicity, we use the L1 Messenger as the test contract.
+contract CrossDomainMessenger_Test is Messenger_Initializer {
+    // Ensure that baseGas passes for the max value of _minGasLimit.
+    function test_baseGas() external {
```

Comment:
I'm curious about the max size of the message before it will revert given max uint32

Reply:
Agree, that's a good question. I'll modify these tests a bit to make sure they're clearly showing where the boundaries are.

Reply:
@tynes I added a test showing that the message length is not bounded by arithmetic overflow/underflow, but rather by the block gas limit, meaning it's not possible to submit a large enough message to cause a revert here.

https://github.com/ethereum-optimism/optimism/blob/10c12d6173ef7028269f218a8debc1038f6ff25f/packages/contracts-bedrock/contracts/test/CrossDomainMessenger.t.sol#L20